src/components/NewGalaxyModal.js

```diff
@@ -7,29 +7,40 @@ import { withModalDrawer } from 'src/components/ModalDrawer'
 import TitleBar from 'src/components/TitleBar'
 import { machineTypes } from 'src/data/machines'
 import { Ajax } from 'src/libs/ajax'
+import { currentApp } from 'src/libs/cluster-utils'
 import colors from 'src/libs/colors'
 import * as Utils from 'src/libs/utils'
 
 
 const styles = {
   whiteBoxContainer: { padding: '1rem', borderRadius: 3, backgroundColor: 'white' },
-  drawerContent: { display: 'flex', flexDirection: 'column', flex: 1, padding: '1.5rem' }
+  drawerContent: { display: 'flex', flexDirection: 'column', flex: '0 1 0%', padding: '1.5rem' }
 }
 
 export const NewGalaxyModal = withModalDrawer({ width: 675 })(class NewGalaxyModal extends Component {
   static propTypes = {
     onDismiss: PropTypes.func.isRequired,
     onSuccess: PropTypes.func.isRequired,
     namespace: PropTypes.string.isRequired,
-    app: PropTypes.object
+    apps: PropTypes.array
```

Comment:
We started pulling in all the runtimes in the PD work for filtering, I think this does a bit of future proofing against that need here, and matches the new pattern

Reply:
smart thinking!